test/unit/middlewares/error-handler.spec.js

```diff
@@ -206,5 +203,34 @@
 					expect(broker.logger.error).toHaveBeenCalledWith(err);
 				});
 		});
+
+		describe('if the error handler resolves the error', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'if·the·error·handler·resolves·the·error'` with `"if·the·error·handler·resolves·the·error"`](https://app.codacy.com/gh/moleculerjs/moleculer/pullRequest?prid=8141835)
```suggestion
		describe("if the error handler resolves the error", () => {
```